### DISCUSSION TITLE
Gmail.py Using Spyder

### DISCUSSION THREAD

**Discussion by Tchaclala** [community/community#33884] [Other Feature Feedback, Questions, & Ideas]
Hi Akashblackthat,

Just wanted to have some indications about your gmailhack.py, stmp.ehlo doesn't work on Spyder (python 3.9) maybe can you indicate me the way to make it working. 

thank you for work :))

Hacker_passionate_of_the_tech

**Comment by cosmicqbit** [comment]
You can search the error on Google, DuckDuckGo or Stackoverflow.
